### DISCUSSION TITLE
Hide Files from Table of Contents

### DISCUSSION THREAD

**Discussion by CubityFirst** [oleeskild/obsidian-digital-garden#180] [Ideas]
There's some notes that I want to be able to publish (for the sake of linking elsewhere, archiving purposes etc, referential notes etc.), but I don't really want them to be indexed on the main page.

I'm not sure how feasible it is, but some sort of frontmatter for "dg-hide" that forces it to not show on the table of contents would be a really nice touch.

Thanks :)

**Comment by oleeskild** [comment]
I agree, that would be a nice setting to have. Just to make sure, when you mention the "Table of Contents", do you mean the file tree on the left hand side, showing the folders and files?

**Comment by CubityFirst** [comment]
Yep! The file tree on the left hand side.

**Comment by CubityFirst** [comment]
Looks like that this was resolved from #228
